jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java

```diff
@@ -1759,6 +1760,36 @@ public void testInputOutputSetsOfIoSpecification() throws Exception {
 
     }
 
+    @Test
+    public void testDisableBpsimDisplayViaProfileSetting() throws Exception {
+        Definitions definitions = loader.loadProcessFromJson("userTask.json", "false", "false");
+        assertEquals(0, definitions.getRelationships().size());
+    }
+
+    @Test
+    public void testDisableBpsimDisplayViaSystemProperty() throws Exception {
+        System.setProperty(EditorHandler.BPSIM_DISPLAY, "false");
+        Definitions definitions = loader.loadProcessFromJson("userTask.json", "false", "true");
+        // should still be 0 even tho "true" is passed for bpsimDisplay
+        assertEquals(0, definitions.getRelationships().size());
+        System.clearProperty(EditorHandler.BPSIM_DISPLAY);
+    }
+
+    @Test
+    public void testEnableBpsimDisplayViaSystemProperty() throws Exception {
+        System.setProperty(EditorHandler.BPSIM_DISPLAY, "true");
+        Definitions definitions = loader.loadProcessFromJson("userTask.json", "false", "false");
```

Comment:
This brings me to idea, that the method `loadProcessFromJson` signature don't need to take also `bpsimdisplay` as an argument, because it can be easily set by system property (as you did). I think the less count of method attributes could help keep code readable, and setting the system property `EditorHandler.BPSIM_DISPLAY` is quite self explanatory enough I think. Let me know what do you think please.